### DISCUSSION TITLE
cd or ls in docker

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#524] [Questions] [answered]
Switching to directories with cd or displaying directory contents with ls does not work. Can someone help me how to do this?

do not work
sudo cd /var/lib/docker/volumes/nextcloud_aio_nextcloud/_data/
sudo ls /var/lib/docker/volumes/nextcloud_aio_nextcloud/data/

**Comment by szaimen** [comment, marked as answer]
Please ask in the forum for this. https://help.nextcloud.com
